kaldb/src/main/java/com/slack/kaldb/bulkIngestApi/BulkIngestApi.java

```diff
@@ -70,6 +73,7 @@ public HttpResponse addDocument(String bulkRequest) {
       }
 
       for (Map.Entry<String, List<Trace.Span>> indexDocs : docs.entrySet()) {
+        incomingDocsTotal.increment(indexDocs.getValue().size());
```

Comment:
adding this helps me debug how many docs does our upstream kafka topic receive and how many docs does logstash forward to Kaldb